api/tests/Api/ECampApiTestCase.php

```diff
@@ -25,6 +25,13 @@
 abstract class ECampApiTestCase extends ApiTestCase {
     use RefreshDatabaseTrait;
 
+    // ensure tests are not run with the assumption, that the server runs on UTC
+    //
+    // can't enable this by default as this breaks several validation messages, however
+    // nothing wrong with enabling this from time to time to ensure we don't mess anything
+    // serious up with DateTime conversion
+    private const USE_NONUTC_TIMEZONE = false;
```

Comment:
Das geht garantiert vergessen, oder wenn wir mal wieder daran denken wissen wir nicht mehr, welche Tests mit diesem Flag failen dürfen und welche nicht.
Wie wäre es wenn wir ein paar einzelne spezifische Tests schreiben die mit einer anderen Zeitzone laufen und die immer Teil unserer normalen Testsuite sind?

Reply:
Der PR https://github.com/ecamp/ecamp3/pull/2373 läuft nun mit allen Tests durch, selbst wenn dieses flag auf true gesetzt ist. 
Ich finde deinen Vorschlag aber auch ok, ein paar spezifische Tests zu schreiben oder spezifische Tests auf eine andere Zeitzone zu switchen. Werde ich im PR #2372 implementieren.

Reply:
Implemented in https://github.com/ecamp/ecamp3/pull/2373/commits/88bd389f8a5b3aba566bded5de5723b645ea8907